CommunicationLayer/PacketDecoder.py

```diff
@@ -23,20 +23,13 @@ def generateInterface(self):
         # Create the header file
         with open(filePath,"w") as file:
             file.write(self.basicHeader)
-            sectionList = []
-            for sectionObj in self.parsedData:
-                sectionName = list(sectionObj.keys())[0]
-                sectionName = re.sub(r'\d+', '', sectionName)
-                if sectionName in sectionList:
-                    continue
-                sectionList.append(sectionName)
```

Comment:
None of this is needed anymore since we can just iterate over the keys of the json object. No need for the regex with MotorDetails now coming through as one object